interfaces/builtin/unity7.go

```diff
@@ -62,9 +62,18 @@ const unity7ConnectedPlugAppArmor = `
 /usr/share/thumbnailer/icons/**            r,
 /usr/share/themes/**                       r,
 
-#owner @{HOME}/.themes/                r,
-#owner @{HOME}/.themes/**              r,
-
+# Snappy's 'xdg-open' talks to the snapd-xdg-open service which currently works
+# only in environments supporting dbus-send (eg, X11). In the future once
+# snappy's xdg-open supports all snaps images, this access may move to another
+# interface.
+/usr/local/bin/xdg-open ixr,
+/usr/bin/dbus-send ixr,
+dbus (send)
+    bus=session
+    path=/
+    interface=com.canonical.SafeLauncher
```

Comment:
This is the interface that is used by snapd-xdg-open in xenial-proposed (ie, the one we should be allowing).